.delivery/build_cookbook/Berksfile

```diff
@@ -3,4 +3,5 @@ source 'https://supermarket.chef.io'
 metadata
 
 cookbook 'expeditor-build',
-         git: 'git@github.com:chef-cookbooks/expeditor-build.git'
+         git: 'git@github.com:chef-cookbooks/expeditor-build.git',
+         branch: 'schisamo/inspec-helpers'
```

Comment:
I'll remove this once we've merged https://github.com/chef-cookbooks/expeditor-build/pull/23. We are using this change to validate the new helpers before merging and risking a breaking of the Automate pipeline.